test/e2e/service.go

```diff
@@ -818,18 +819,20 @@ var _ = Describe("Services", func() {
 			}()
 		}
 
-		By("creating service " + serviceName + " with load balancer in namespace " + ns2)
-		t2 := NewWebserverTest(c, ns2, serviceName)
+		By("creating service " + serviceName + " with UDP load balancer in namespace " + ns2)
+		t2 := NewNetcatTest(c, ns2, serviceName)
 		svc2 := t2.BuildServiceSpec()
 		svc2.Spec.Type = api.ServiceTypeLoadBalancer
 		svc2.Spec.Ports[0].Port = servicePort
+		// Let this one be UDP so that we can test that as well without an additional test
```

Comment:
please reword comment to something like 'udp loadbalancing is tested via test X'